UKHomeOffice/nmsw-ui
# Bug fix for FAL 5 and 6 download on file upload pages

# Ticket

N/A

----

## AC

After downloading FAL5 and FAL6 forms from upload FAL5 and FAL6 page and trying to open it, we were getting error that file could be corrupted - this was because we were missing a '/' from the import

----

## To test

- go to Crew upload page
- download the Crew details form file
- > it should open correctly
- repeat for Passenger form file

----

## Notes
